t/info.t

```diff
@@ -75,33 +75,79 @@ is_json_GET(
     return $info_species;
   };
 
-  my $expected = {species => [ { division => 'EnsemblVertebrates', name => 'homo_sapiens', 'accession' => 'GCA_000001405.9', common_name => 'human', display_name => 'Human', taxon_id => '9606', groups => ['core', 'funcgen', 'variation'], aliases => [], release => $core_schema_version, assembly => 'GRCh37', strain => 'test_strain', strain_collection => 'human'} ]};
-  my $expected_hide_strain_info = {species => [ { division => 'EnsemblVertebrates', name => 'homo_sapiens', 'accession' => 'GCA_000001405.9', common_name => 'human', display_name => 'Human', taxon_id => '9606', groups => ['core', 'funcgen', 'variation'], aliases => [], release => $core_schema_version, assembly => 'GRCh37'} ]};
+  my $expected = {
+    species => [
+      {
+        division => 'EnsemblVertebrates',
+        name => 'homo_sapiens',
+        accession => 'GCA_000001405.9',
+        common_name => 'human',
+        display_name => 'Human',
+        taxon_id => '9606',
+        groups => ['core', 'funcgen', 'variation'],
+        aliases => [],
+        release => $core_schema_version,
+        assembly => 'GRCh37',
+        strain => 'test_strain',
+        strain_collection => 'human'
+      }
+    ]
+  };
+  my $expected_hide_strain_info = {
+    species => [
+      { 
+        division => 'EnsemblVertebrates',
+        name => 'homo_sapiens',
+        accession => 'GCA_000001405.9',
+        common_name => 'human',
+        display_name => 'Human',
+        taxon_id => '9606',
+        groups => ['core', 'funcgen', 'variation'],
+        aliases => [],
+        release => $core_schema_version,
+        assembly => 'GRCh37'
+      }
+    ]
+  };
   my $expected_empty_list = {species => [ ]};
 
   eq_or_diff_data(
-    $get_species->('/info/species', 'Checking only DBA available is the test DBA'), $expected, 
-    "/info/species | Checking only DBA available is the test DBA");
+    $get_species->('/info/species', 'Checking only DBA available is the test DBA'),
+    $expected, 
+    "/info/species | Checking only DBA available is the test DBA"
+  );
   eq_or_diff_data(
-    $get_species->('/info/species?division=EnsemblVertebrates', q{Output is same as /info/species if specified 'EnsemblVertebrates' division}), $expected, 
-    "/info/species?division=EnsemblVertebrates | Output is same as /info/species if specified 'Ensembl' division");
+    $get_species->('/info/species?division=EnsemblVertebrates', q{Output is same as /info/species if specified 'EnsemblVertebrates' division}),
+    $expected, 
+    "/info/species?division=EnsemblVertebrates | Output is same as /info/species if specified 'Ensembl' division"
```

Comment:
Shouldn't the message be "......if specified 'EnsemblVertebrates' division" ?!

Reply:
Agreed